src/custom-lists/selectors.js

```diff
@@ -0,0 +1,104 @@
+import { createSelector } from 'reselect'
+
+import { selectors as filters } from 'src/overview/filters'
+
+// TODO: Needs some work.
+const sortAlphabetically = (a, b) => {
+    if (
+        typeof a === 'string' &&
+        typeof b === 'string' &&
+        a.name.toLowerCase() < b.name.toLowerCase()
```

Comment:
These conditions don't make sense. The `typeof a === 'string'` is known as a "type guards" meaning, if they pass, later uses of that name in the condition should assume the type they were tested against. Hence, when used in line 10 they should be assumed as JS type `string` (most modern editors should show this if set up right), but then you're accessing the property `name` which doesn't existing on `string` typed values.

I think either you should be checking `a.name` and `b.name` are `string` type (instead of `a` and `b`), or (preferably) skipping these type guards in these conditions and ensuring that the values passed into this function are the type you need.

Reply:
They were just temporary for the dummy data earlier, don't need them now.